### DISCUSSION TITLE
CssSyntaxError dont let editor load

### DISCUSSION THREAD

**Discussion by capoia** [GrapesJS/grapesjs#5031] [Q&A]
Hello, I use grapesjs to make landing pages, in the end I save everything that needs to be saved in the database (html, css, components, etc.) and then when I open the editor I retrieve all this info and start grapesjs.
The problem is, sometimes a user edits the landing page and ends up forgetting silly things, like not putting the semicolon.
The ideal thing would be for the editor to load anyway, for the person to correct it, but it can't even load, since it generates an exception, how would I go about solving this problem?
![CleanShot 2023-04-05 at 10 17 48](https://user-images.githubusercontent.com/21159928/230092303-15102f70-0fc3-4d9a-b908-ca8d95f11ce9.png)
![CleanShot 2023-04-05 at 10 20 20](https://user-images.githubusercontent.com/21159928/230092903-e57988fd-18b2-48fb-9d78-0809013d8856.png)

**Comment by artf** [comment]
Can you elaborate on how the user is able to add invalid CSS?

**Reply by capoia** [reply]
What I'm sure causes this is incomplete css within a custom code.
But I'm still trying to find out what causes this, because I have reports of people who are using it (I built a saass around it) that only mess with the interface (changing image, font color etc...) which also generated this error, but I haven't been able to simulate it yet.
But anyway, it would be interesting to point out the error, but let it edit, after all, currently when a person does something wrong with css and leaves the pagebuilder and returns to it, the exception doesn't even let him in to edit.
